admin/sql/create_foreign_keys.sql

```diff
@@ -2,21 +2,21 @@ BEGIN;
 
 ALTER TABLE token
   ADD CONSTRAINT token_owner_id_fkey FOREIGN KEY (owner_id)
-  REFERENCES "user" (id) MATCH SIMPLE
+  REFERENCES "supporter" (id) MATCH SIMPLE
```

Comment:
we could finally get rid of the " " since user is not a reserved name, right?

Reply:
yup. done